app/components/article-content.js

```diff
@@ -388,6 +390,36 @@ export default Component.extend(
 			}
 		},
 
+		/**
+		 * Handles opening sections when click event occurs on reference
+		 *
+		 * @returns {void}
+		 */
+		handleReferences(event) {
+			if (event.target.closest('a[href^="#cite_ref-"]')) {
+				const element = event.target.closest('a[href^="#cite_ref-"]'),
+					currentCiteNote = document.getElementById(element.hash.slice(1, element.length)),
+					closest = currentCiteNote.closest('section[id*="section"]');
+
+				if (closest) {
+					let currentHeader = document.getElementById(closest.previousElementSibling.id);
```

Comment:
getElementById can return null

@bkoval maybe we should write a small wrapper for common getters?